xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -36,10 +36,12 @@ using namespace PVR;
 using namespace EPG;
 
 CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
-  CPVRChannelGroup(bRadio, bRadio ? PVR_INTERNAL_GROUP_ID_RADIO : PVR_INTERNAL_GROUP_ID_TV, g_localizeStrings.Get(19287)),
+  CPVRChannelGroup(),
```

Comment:
Base class constructors without arguments are called automatically. This line can be removed.